test/lib/sdkdriver/test/unit/spec/transport/websocket/server.ts

```diff
@@ -0,0 +1,152 @@
+import WSServerTransport from '../../../../../src/transport/websocket/server';
+import { WebSocket } from '../../../mocks';
+import * as assert from 'assert';
+import * as sinon from 'sinon';
+
+describe('WSServerTransport', () => {
+  describe('constructor', () => {
+    let transport: WSServerTransport;
+
+    it('should create an underlying WebSocket server with the given port', () => {
+      transport = new WSServerTransport(2000, { WebSocket });
+      sinon.assert.calledWith(WebSocket.Server, { port: 2000 });
+    });
+  });
+
+  describe('#close', () => {
+    let closeEventPromise: Promise<void>;
+    let transport: WSServerTransport;
+    let wsConnection: WebSocket;
+    let wsServer: WebSocket.Server;
+
+    before(async () => {
+      transport = new WSServerTransport(2000, { WebSocket });
+      const openPromise: Promise<void> = transport.open();
+      wsConnection = new WebSocket('foo');
+      wsServer = transport._wsServer;
```

Comment:
I feel similarly about accessing `_wsServer` as I do `_wsClient` in the other tests.